### ISSUE TITLE
Source not able to build

### ISSUE THREAD

**Issue by DaveG9856** [PocketMobsters/MobBotTemp#39]
We need a fix to the source to remove the build errors.Or at least a copy/paste guide to fix them.

Severity	Code	Description	Project	File	Line	Suppression State
Error	CS0246	The type or namespace name 'DankMemes' could not be found (are you missing a using directive or an assembly reference?)	PokemonGo.RocketAPI	J:\pokebot\source\MobBotTemp\Pokemon-Go-Rocket-API\PokemonGo.RocketAPI\Login\GoogleLogin.cs	1	Active

Error	CS0006	Metadata file 'J:\pokebot\source\MobBotTemp\Pokemon-Go-Rocket-API\PokemonGo.RocketAPI\bin\Debug\Pokemon.Go.Rocket.API.dll' could not be found	PoGo.PokeMobBot.Logic	J:\pokebot\source\MobBotTemp\PoGo.PokeMobBot.Logic\CSC	1	Active

Error	CS0006	Metadata file 'J:\pokebot\source\MobBotTemp\PoGo.PokeMobBot.Logic\bin\Debug\PoGo.PokeMobBot.Logic.dll' could not be found	PoGo.PokeMobBot.CLI	J:\pokebot\source\MobBotTemp\PoGo.PokeMobBot.CLI\CSC	1	Active

Error	CS0006	Metadata file 'J:\pokebot\source\MobBotTemp\Pokemon-Go-Rocket-API\PokemonGo.RocketAPI\bin\Debug\Pokemon.Go.Rocket.API.dll' could not be found	PoGo.PokeMobBot.CLI	J:\pokebot\source\MobBotTemp\PoGo.PokeMobBot.CLI\CSC	1	Active

**Comment by kraleksandr** [comment]
Replace your PokemonGo.RocketAPI.csproj with this code
http://pastebin.com/tqYXZ4bm

**Comment by DaveG9856** [comment]
nope same errors :(

Severity	Code	Description	Project	File	Line	Suppression State
Error	CS0006	Metadata file 'J:\pokebot\source\MobBotTemp\Pokemon-Go-Rocket-API\PokemonGo.RocketAPI\bin\Debug\Pokemon.Go.Rocket.API.dll' could not be found	PoGo.PokeMobBot.Logic	J:\pokebot\source\MobBotTemp\PoGo.PokeMobBot.Logic\CSC	1	Active
Error	CS0227	Unsafe code may only appear if compiling with /unsafe	PokemonGo.RocketAPI	J:\pokebot\source\MobBotTemp\Pokemon-Go-Rocket-API\PokemonGo.RocketAPI\Helpers\RequestBuilder.cs	198	Active
Error	CS0006	Metadata file 'J:\pokebot\source\MobBotTemp\PoGo.PokeMobBot.Logic\bin\Debug\PoGo.PokeMobBot.Logic.dll' could not be found	PoGo.PokeMobBot.CLI	J:\pokebot\source\MobBotTemp\PoGo.PokeMobBot.CLI\CSC	1	Active
Error	CS0006	Metadata file 'J:\pokebot\source\MobBotTemp\Pokemon-Go-Rocket-API\PokemonGo.RocketAPI\bin\Debug\Pokemon.Go.Rocket.API.dll' could not be found	PoGo.PokeMobBot.CLI	J:\pokebot\source\MobBotTemp\PoGo.PokeMobBot.CLI\CSC	1	Active

**Comment by DaveG9856** [comment]
not worried about the unsafe error that I can fix, the dll's are missing and the whole folder is empty

**Comment by DaveG9856** [comment]
all fixed

**Comment by Dominiol** [comment]
I have the same problem. There are some missin dll's.

**Comment by permag02** [comment]
what is the fix?

**Comment by DaveG9856** [comment]
step #1 right click on PoGo.PokeMobBot.CLI 
select properties
goto build
check allow unsafe code
do this for each project

Step #2 right click on PokemonGo.RocketAPI
manage nuget packages
click output tab look for missing package name 
find that name in the nuget list 
hit update

do this for each error
if it still doesn't work make sure you have the correct version # 

this is the best I can explain it sorry if it doesn't help.

**Comment by Dominiol** [comment]
Yeah, [it worked for me](https://github.com/PocketMobsters/MobBotTemp/issues/42)
But thank you very much!

**Comment by littlemonster1** [comment]
I am still having this issue even after doing both steps (at least to the best of my knowledge)

**Comment by jjskuld** [comment]
Dup of #85
